react/introduction/introduction_to_react.md

```diff
@@ -60,4 +59,4 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-- If you are interested in learning more about React's history, check out [this documentary on React's origin story by Honeypot](https://www.youtube.com/watch?v=8pDqJVdNa44)
+- check out this [documentary on React’s origin story by Honeypot](https://www.youtube.com/watch?v=8pDqJVdNa44).
```

Comment:
```suggestion
- If you are interested in learning more about React's history, check out this [documentary on React's origin story by Honeypot](https://www.youtube.com/watch?v=8pDqJVdNa44).
```
Think you accidentally chopped off the first part of the sentence. The original text to replace only started from the `check out ...` part, leaving the first bit of the sentence alone.